core/config/project_settings.cpp

```diff
@@ -510,19 +510,19 @@ Error ProjectSettings::_setup(const String &p_path, const String &p_main_pack, b
 Error ProjectSettings::setup(const String &p_path, const String &p_main_pack, bool p_upwards) {
 	Error err = _setup(p_path, p_main_pack, p_upwards);
 	if (err == OK) {
-		String custom_settings = GLOBAL_DEF("application/config/project_settings_override", "");
+		String custom_settings = PROJECT_DEFAULT("application/config/project_settings_override", "");
```

Comment:
GLOBAL_DEF is not short for GLOBAL_DEFAULT, its GLOBAL_DEFINE.

Reply:
I suppose this is a good argument for renaming it then.

Reply:
Haha I always thought it was define, not default, and was a bit confused reading the OP.